src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss

```diff
@@ -18,8 +22,27 @@
   display: flex;
   flex-shrink: 0;
   align-items: center;
+  opacity: 0;
+  @include backgroundGradient($hover-ui);
 }
 
-.#{$prefix}--data-table tbody tr:hover .#{$iot-prefix}--row-actions-container__background {
-  background: linear-gradient(90deg, rgba(229, 229, 229, 0) 0, #{$hover-ui} 16px);
+.#{$prefix}--data-table tbody tr:hover .#{$iot-prefix}--row-actions-container__background,
+.#{$iot-prefix}--row-actions-container__background--overflow-menu-open {
+  opacity: 1;
```

Comment:
```suggestion
  opacity: 1;
  transition: opacity $duration--fast-02 motion(entrance, productive);
```

Reply:
I believe this will help with the slight visibility we see on hover for the overflow.

Reply:
Great, thanks :-)